### DISCUSSION TITLE
No matching profiles found for bundle identifier

### DISCUSSION THREAD

**Discussion by NickWoodUK** [codemagic-ci-cd/codemagic-docs#1068] [Q&A] [answered]
Hello.

I am trying to do a build and i get the following: No matching profiles found for bundle identifier 

Any ideas?

Thanks

**Comment by himesh-cm** [comment, marked as answer]
HI, @NickWoodUK. Can you confirm if the bundle identifier in the App store account is the same as your Xcode Project?

**Reply by NickWoodUK** [reply]
Resolved. Thank you!
